<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(85, 85, 127)</string>
  </property>
  <widget class="QRadioButton" name="radioRed">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Led vermelho</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioGreen">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Led verde</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioYellow">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Led amarelo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEnviar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(0, 170, 255);</string>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
